CHANGELOG.md

```diff
@@ -27,7 +27,14 @@ Example:
   });
   ```
 
-Note that this option is incompatible with connect option `maxVideoMaxBitrate`, and will result in an error at connect time if specified along with the new options.
+Additional notes:
+- This feature is incompatible with connect option `maxVideoBitrate`, and will result in an error at connect time if specified along with the new option.
```

Comment:
I think this should be moved up higher so this section can be focused on known limitations. This is more instructions for use. I would add the following known limitation:

"The beta currently does not support setting a max bitrate for your simulcast layers."